index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<header>
+    <div class="name">John Doe</div>
+    <div class="menu"><img src="images/menu.svg" alt="menu"></div>
+</header>
+<div class="container">
+    <div class="hero">
+        <h1>Hey There. <br>I’m Alex</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <ul>
+            <li><a href=""><img src="images/links/github.svg" alt="github"></a></li>
+            <li><a href=""><img src="images/links/linkedin.svg" alt="linkedin"></a></li>
+            <li><a href=""><img src="images/links/angelist.svg" alt="angelist"></a></li>
+            <li><a href=""><img src="images/links/twitter.svg" alt="twitter"></a></li>
+            <li><a href=""><img src="images/links/medium.svg" alt="medium"></a></li>
+        </ul>
+        <div class="illustration"><img src="images/illustration-header-mobile.svg" alt="illustration"></div>
+    </div>
+</div>
```

Comment:
- [ ] Kindly make sure to wrap this headline part into the `section` tag. It will increase the website accessibility and we will maintain the correct usage of semantics 😉 Please take a look at [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) to fully understand the reason behind it.